configs/templates/generated_interfaces_for_adapters/list/ListProcessor.go

```diff
@@ -0,0 +1,14 @@
+package list
+
+import "istio.io/mixer/configs/templates/list"
+
+type Instance struct {
+  Blacklist       bool
+  CheckExpression interface{}
+}
+
+type ListProcessor interface {
+  ConfigureList(templateName string, types map[string]*foo_bar_mylistchecker.Type /*typeName to Type mapping*/) error
+  CheckList(templateName string, instances map[string]*Instance /*typeName to Instance (generated from Constructor) mapping*/) (bool, error)
```

Comment:
And IMO it's fine to mix protos and golang structs - from the implementer's perspective they're working with golang structs in either case.